src/poprox_recommender/model/__init__.py

```diff
@@ -24,6 +29,13 @@ class ModelConfig:
     pretrained_model = "distilbert-base-uncased"
 
 
+@dataclass
+class RecommenderComponents:
+    tokenizer: AutoTokenizer
+    model: NRMS
```

Comment:
Curious if we need a more general class for the model object (like a parent class for both NRMS, PFAR, etc.). But otherwise looks good!

Reply:
Eventually, yes, for now just want to capture what we're currently doing as a minimal increment.

Reply:
Once @karlhigley has gotten farther on the refactor + pipelining I expect this class will go away and we'll just have a function that returns the whole pipeline.